PayloadPkg/FirmwareUpdate/FirmwareUpdateHelper.c

```diff
@@ -431,11 +434,165 @@ UpdateContainerComp (
   // Component base = Container base + data offset from container base + offset of component inside container
   //
   ComponentBase = ContainerEntryPtr->Base + ContainerHdr->DataOffset + ComponentEntryPtr->Offset;
+
+  // Check Svn for container component
+  FlashCompLzHeader = (LOADER_COMPRESSED_HEADER *) (UINTN) ComponentBase;
+  CapCompLzHeader   = (LOADER_COMPRESSED_HEADER *) ((UINTN)ImageHdr + sizeof(EFI_FW_MGMT_CAP_IMAGE_HEADER));
+  if (CapCompLzHeader->Svn < FlashCompLzHeader->Svn) {

```

Comment:
Add check for LZ signature to ensure it is a LZ header.

Reply:
Added "IsLzHeader" to LZ signature.